anta/input_models/interfaces.py

```diff
@@ -7,17 +7,33 @@
 
 from typing import Literal
 
-from pydantic import BaseModel
+from pydantic import BaseModel, ConfigDict
 
-from anta.custom_types import Interface
+from anta.custom_types import Interface, PortChannelInterface
 
 
 class InterfaceState(BaseModel):
     """Model for an interface state."""
 
+    model_config = ConfigDict(extra="forbid")
     name: Interface
     """Interface to validate."""
-    status: Literal["up", "down", "adminDown"]
-    """Expected status of the interface."""
+    status: Literal["up", "down", "adminDown"] | None = None
+    """Expected status of the interface. Required field in the `VerifyInterfacesStatus` test."""
     line_protocol_status: Literal["up", "down", "testing", "unknown", "dormant", "notPresent", "lowerLayerDown"] | None = None
     """Expected line protocol status of the interface."""
+    portchannel: PortChannelInterface | None = None
+    """Port Channel in which the interface is bundled. Required field in the `VerifyLACPInterfacesStatus` test."""
```

Comment:
In EOS they put `Port-Channel` with an hyphen:
```
interface Port-Channel11
   description MLAG_DC1-LEAF1B_Port-Channel11
   switchport mode trunk
   switchport trunk group MLAG
```
```suggestion
    """Port-Channel in which the interface is bundled. Required field in the `VerifyLACPInterfacesStatus` test."""
```

Reply:
Updated.